features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/netflow5/proto/Packet.java

```diff
@@ -70,6 +75,18 @@ public Packet(final Header header,
         this.records = records;
     }
 
+    private FlowMessageProto.Builder createFlowMessage(Header header, Record record) {
```

Comment:
I would prefer to have these messages in the parser and keep the proto implementations clean of that knowledge.